.github/workflows/chromatic-main.yaml

```diff
@@ -1,21 +1,14 @@
 # Chromatic action from https://www.chromatic.com/docs/github-actions
 
-name: "chromatic"
```

Comment:
Do we need to distinguish the naming? These aren't going to run in the same check list right?

Reply:
Yeah I was wondering about that. Technically they shouldn't, but just in case? 😬 Also just to be explicit that this one is only for `main`, although I guess I could add a description instead 🤔